bitrise.yml

```diff
@@ -571,8 +571,23 @@ workflows:
                     filesChanged=`git diff --name-only $GIT_CLONE_COMMIT_HASH $COMMIT_ON_MAIN_PR`
                     echo $filesChanged
                 fi
-
-                doNotRunTests='^(\.[a-zA-Z]+\.y*|Docs/*|Sync*/*|.github/|[a-zA-Z]+\.md|[a-zA-Z]+\.sh|[a-zA-Z0-9]+[-_]+[a-zA-Z]+\.sh)|fastlane/*|taskcluster/*|L10nSnapshotTests/*|Shared/[a-zA-Z]+\.lproj/*|Shared/[A-Za-z]/[a-zA-Z]+\.lproj/*|WidgetKit/[a-zA-Z]+\.lproj/*|Client/*/[a-zA-Z]+\.lproj/*)'
+                # Match on file names in all directories and subdirectories
+                #
+                # Project Configs
+                #   .*[^bitrise]\.y[^/]*$ -> matches all files not named bitrise whose file extension begins with a 'y'
+                #   .*\.sh$ -> matches all files whose file extension begins with 'sh'
+                #   .*\.md$ -> matches all files whose file extension begins with 'md'
+                #   .*\.lproj/.* -> matches all files whose parent directory begins with 'lproj'
+                #   fastlane/* -> matches all files in the fastlane directory
+                #   taskcluster/* -> matches all files in the taskcluster directory
+                #
+                # Features tested outside bitrise
+                #   Sync*/* -> matches all files in the Sync,SyncIntegrationTests,SyncTelemetry directories
+                #   L10nSnapshotTests/* -> matches all files in the L10nSnapshotTests directory
+                #
+                # Test Helpers
+                #   test-fixtures/.py$ -> matches all files whose file extension begine with 'py'
```

Comment:
For test-fixtures, if the file is an .html file we should run the tests. Just double checking that that's the case here :) Thanks!

Reply:
yep! just included the `.py` files for now and didn't want to include any of the `.db` or `.html` files just in case.